ts/components/PinCreationForm.tsx

```diff
@@ -40,14 +46,16 @@ export const PinCreationForm = ({ onSubmit }: Props) => {
   const [isPinConfirmationDirty, setIsPinConfirmationDirty] =
     React.useState(false);
 
-  const isPinValid = !isPinDirty || pin.length === pinLength;
+  const isPinValid =
+    !isPinDirty ||
+    isValidPinNumber(pin) ||
+    (isDevEnv && pin.length === pinLength);
 
   const isPinConfirmationValid =
     !isPinConfirmationDirty || (pinConfirmation && pinConfirmation === pin);
 
   const isFormValid =
-    pin.length === pinLength &&
-    pinConfirmation.length === pinLength &&
+    (isValidPinNumber(pin) || (isDevEnv && pin.length === pinLength)) &&
```

Comment:
ditto

Reply:
What do you think about a button placed at the bottom that insert a valid pin only in Dev mode?![image](https://github.com/pagopa/io-app/assets/114305019/6fadaafe-ba54-40a4-82dc-5882b361938e)

Reply:
This could be a good way to speed up the pin creation in dev mode but:
- How can I know the chosen pin to enter it during identification? Entering an "easy" pin is more important to speed up how we exit the identification screen I guess.
- If we want to add this (that could be a good idea) we should add a way to speed up the identification phase too.

What do you think?

Reply:
In addition to the button on the pin creation screen, there will be a button on the lock screen that inserts the same pin.
I was waiting for a confirmation before proceeding with the implementation in the lock screen too :slightly_smiling_face:

Reply:
LGTM ;)